pkg/kube/controllers/quarkssecret/quarkssecret_reconciler.go

```diff
@@ -619,56 +626,89 @@ func (r *ReconcileQuarksSecret) createSecrets(ctx context.Context, qsec *qsv1a1.
 	// Create the main secret
 	// Check if allowed to generate secret, could be already done or
 	// created manually by a user
-	skipCreation, err := r.skipCreation(ctx, qsec)
+
+	userCreatedSecret := false
+	skipCreation, existingSecret, err := r.skipCreation(ctx, qsec)
 	if err != nil {
 		ctxlog.Errorf(ctx, "Error reading the secret: %v", err.Error())
 	}
+
 	if skipCreation {
 		ctxlog.WithEvent(qsec, "SkipCreation").Infof(ctx, "Skip creation: Secret '%s/%s' already exists and it's not generated", qsec.Namespace, qsec.Spec.SecretName)
+		secret = existingSecret
+
+		// Add user created annotation
+		if secret != nil {
+			if secret.Annotations == nil {
+				secret.Annotations = map[string]string{}
+			}
+			secret.Annotations[qsv1a1.AnnotationUserCreatedSecret] = "true"
```

Comment:
What is this used for? I can't find code that references it.

There is also the `quarks.cloudfoundry.org/secret-kind=generated` label.

Reply:
True. I forgot to remove this part of the code from the PR.